app/models/container_label_tag_mapping/mapper.rb

```diff
@@ -0,0 +1,90 @@
+class ContainerLabelTagMapping
+  # Performs most of the work of ContainerLabelTagMapping - holds current mappings,
+  # computes applicable tags, and creates/finds Tag records - except actually [un]assigning.
+  class Mapper
```

Comment:
@cben could you add YARD doc format, at least for the public methods? Can be theoretically done in another PR, but a new class is a nice opportunity. :-)

Reply:
Sure :+1:  Learning it now...